app/views/ControlWindow.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * @author Bingzheng Feng
+ * @module views/ControlWindow
+ * @requires models/Commands
+ * @requires util/Validator
+ * @requires util/Logger
+ * @requires models/TelemetryData
+ * @requires util/Template
+ * @requires electron
+ * @listens models/TelemetryData~TelemetryData:data_received
+ * @copyright Waterloo Aerial Robotics Group 2016
+ * @licence https://raw.githubusercontent.com/UWARG/WARG-Ground-Station/master/LICENSE
+ * @description Responsible for displaying control pitch, yaw, roll
+ */
+
+var remote = require('electron').remote;
+var Template = require('../util/Template');
+var TelemetryData = remote.require('./app/models/TelemetryData');
+var Logger = remote.require('./app/util/Logger');
+var Validator = require('../util/Validator');
+var Commands = remote.require('./app/models/Commands');
+var Channel_Conf = remote.require('./config/vehicle-config');
+var PicPilot_Conf = remote.require('./config/picpilot-config');
+
+module.exports = function (Marionette) {
+
+  return Marionette.ItemView.extend({
+    template: Template('ControlWindow'),
+    className: 'controlWinodw',
+
+    ui: {
+      front23: '.controlWindow.circle.front.twothree',
+      front14: '.controlWindow.circle.front.onefour',
+      ch1_text: '.controlWindow.value.one',
+      ch2_text: '.controlWindow.value.two',
+      ch3_text: '.controlWindow.value.three',
+      ch4_text: '.controlWindow.value.four',
+      autoPilot_btn: '.controlWindow.auto-pilot',
```

Comment:
keep the variable naming convnetion consistent (dont mix camelCase with snake_case. Keep variables to snake_case only